config/api.yml

```diff
@@ -2336,7 +2336,17 @@
         - ems_infra_show_list
         - ems_block_storage_show_list
       - :name: create
-        :identifier: ems_infra_new
+        :identifiers:
+        - :klass: EmsInfra
+          :identifier: ems_infra_new
+        - :klass: EmsCloud
+          :identifier: ems_cloud_new
+        - :klass: Provider #EmsContainer
```

Comment:
Unlike ems_network, we don't have an alias in core for this. It'd be cool if we did, I guess I'm about to find out if reviewers hate it or not. The alternative that's here now I don't like because it doesn't keep a one-to-one mapping in core for product identifiers.